k8s/production/custom/delete-stale-branches/cron-jobs.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: delete-stale-branches
+  namespace: custom
+spec:
+  schedule: "0 4 * * 0"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      activeDeadlineSeconds: 3600 # terminate any running job after 1 hour
+      template:
+        spec:
+          restartPolicy: Never
+          containers:
+          - name: delete-stale-branches
+            image: ghcr.io/spack/gitlab-delete-stale-branches:0.0.1
+            imagePullPolicy: IfNotPresent
+            resources:
+              requests:
+                cpu: 500m
+                memory: 50M
```

Comment:
This memory value seems pretty small, I could see it using more than this easily. Maybe something like 200M would be better? What do you think?

Reply:
For the record I know it's not a limit, but this is more of a scheduling concern (making sure there's enough memory for all containers on a given node).